components/layout/fragment_tree/box_fragment.rs

```diff
@@ -202,14 +197,23 @@ impl BoxFragment {
 
     /// Get the scrollable overflow for this [`BoxFragment`] relative to its
     /// containing block.
-    pub fn scrollable_overflow(&self) -> PhysicalRect<Au> {
+    pub fn scrollable_overflow(&self) -> Option<PhysicalRect<Au>> {
```

Comment:
The `expect` call is the same everywhere this is called right? There is no error handling possible, so all these functions on `Fragment` and `BoxFragment` should probably just call `.expect(...)` internally. That way the callers don't have to worry about what to do if the value is `None` -- which should never happen.